Trying:
    say_my_name = __import__('3-say_my_name').say_my_name
Expecting nothing
ok
Trying:
    say_my_name()
Expecting:
    Traceback (most recent call last):
    ...
    ValueError:
**********************************************************************
File "./tests/3-say_my_name.txt", line 5, in 3-say_my_name.txt
Failed example:
    say_my_name()
Expected:
    Traceback (most recent call last):
    ...
    ValueError:
Got:
    Traceback (most recent call last):
      File "/usr/lib/python3.11/doctest.py", line 1351, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest 3-say_my_name.txt[1]>", line 1, in <module>
        say_my_name()
    TypeError: say_my_name() missing 1 required positional argument: 'first_name'
Trying:
    say_my_name("Hatim", "Selmun")
Expecting:
    My name is Hatim Selmun
ok
Trying:
    say_my_name(12, "Selmun")
Expecting:
    Traceback (most recent call last):
    ...
    TypeError: first_name must be a string
ok
Trying:
    say_my_name("Hatim", 24)
Expecting:
    Traceback (most recent call last):
    ...
    TypeError: last_name must be a string
ok
Trying:
    say_my_name("Hatim")
Expecting:
    My name is Hatim 
ok
**********************************************************************
1 items had failures:
   1 of   6 in 3-say_my_name.txt
6 tests in 1 items.
5 passed and 1 failed.
***Test Failed*** 1 failures.
